src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.controller.js

```diff
@@ -159,7 +159,7 @@ function listViewController($scope, $routeParams, $injector, $timeout, currentUs
         allowBulkPublish: $scope.entityType === 'content' && $scope.model.config.bulkActionPermissions.allowBulkPublish,
         allowBulkUnpublish: $scope.entityType === 'content' && $scope.model.config.bulkActionPermissions.allowBulkUnpublish,
         allowBulkCopy: $scope.entityType === 'content' && $scope.model.config.bulkActionPermissions.allowBulkCopy,
-        allowBulkMove: $scope.model.config.bulkActionPermissions.allowBulkMove,
+        allowBulkMove: $scope.entityType !== 'member' && $scope.model.config.bulkActionPermissions.allowBulkMove,
```

Comment:
@kjac The `AllowBulkMove` value comes from the data type configuration of list views. Maybe this would be better set to false in c# when the data is being built up for this in the member controller or during model mapping, that way we're not hard coding things?